### PR TITLE
[Outlook] (sideloading) Reorganize instructions

### PR DESCRIPTION
**Pull request by samantharamon**
- Cleans up the 'Sideload manually' section now that the add-in entry points have changed.
- Adds a 'Locate a sideloaded add-in' section.

### CODE REVIEW

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 1]
```diff
@@ -30,37 +33,28 @@ If you created your Outlook add-in using the [Yeoman generator for Office Add-in
...
-
-To manually sideload an add-in in the new Outlook on Windows (preview), see [Sideload an add-in in the modern Outlook on the web or new Outlook on Windows (preview)](#modern-outlook-on-the-web-and-new-outlook-on-windows-preview).
+> The sideloading guidance in this section only applies to Outlook add-ins that use an XML manifest. To learn about sideloading options for Outlook add-ins that use the unified manifest for Microsoft 365, see the "Conversion tools and options" section of [Convert an add-in to use the unified manifest for Microsoft 365](../develop/convert-xml-to-json-manifest.md#conversion-tools-and-options).
 
-### Modern Outlook on the web and new Outlook on Windows (preview)
+# [Windows (classic and new) and Web](#tab/windows-web)
 
-1. Go to [Outlook on the web](https://outlook.office365.com) or open the [new Outlook client on Windows (preview)](one-outlook.md).
+Add-ins are manually sideloaded through the **Add-Ins for Outlook** dialog. To sideload an add-in in Outlook on Windows, on the web, and on [new Outlook on Windows (preview)](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627), perform the following steps.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:41]:
```suggestion
Add-ins are manually sideloaded through the **Add-Ins for Outlook** dialog. To sideload an add-in in Outlook on Windows, on the web, and in [new Outlook on Windows (preview)](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627), perform the following steps.
```

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 2]
```diff
@@ -167,7 +79,7 @@ To manually sideload an add-in in the new Outlook on Windows (preview), see [Sid
 
     ![The Get Add-ins button is selected in Outlook on Mac.](../images/outlook-sideload-mac-classic.png)
 
-    > [!IMPORTANT]
+    > [!TIP]
     > If you don't see **Get Add-ins** in your version of Outlook, select the ellipsis button (`...`) from the ribbon, then select **Get Add-ins**.
     >
     > ![The Get Add-ins option is selected from the ellipsis button in Outlook on Mac.](../images/outlook-sideload-new-mac.png)
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:85]: The image in step 4 (and the other tab's step 3) says the dialog title is "Custom Addins". We should update to match. Also, do they plan to fix that in the UI?

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 3]
```diff
@@ -30,37 +33,28 @@ If you created your Outlook add-in using the [Yeoman generator for Office Add-in
...
+    >
+    >   ![Header of the classic Outlook on the web toolbar with "Office 365 | Outlook" as its title.](../images/outlook-on-the-web-classic-toolbar.png)
     >
-    > ![The Get Add-ins option is selected from the ellipsis button in Outlook on the web.](../images/outlook-sideload-simplified-ribbon-web.png)
+    >   To access the **Add-Ins for Outlook** dialog, open [Outlook on the web](https://outlook.office365.com). Then, select the gear icon from the top-right section of the toolbar and choose **Manage add-ins**.
     >
-    > If you don't see the **Get add-ins** option in either the classic or simplified ribbon layouts, you must access the **Add-Ins for Outlook** dialog through <https://aka.ms/olksideload>.
+    >   ![The Manage add-ins option is selected in classic Outlook on the web.](../images/outlook-sideload-web-manage-integrations.png)
+    >
+    >   Note that your organization may include its own logo in the mailbox toolbar, so you might see something slightly different than shown in the preceding images.
```
Comment (Rick-Kirkham) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:55]: Just a suggestion:

```suggestion
    >   Note that your organization may include its own logo in the mailbox toolbar, so you might see something slightly different from what is shown in the preceding images.
```

**Review by Rick-Kirkham — APPROVED** [review]
Approving with just one minor suggestion

File: docs/outlook/sideload-outlook-add-ins-for-testing.md (hunk 2 above)
Comment (samantharamon) [reply docs/outlook/sideload-outlook-add-ins-for-testing.md:85]: Fixed. I'm not sure if it's on their backlog to get it fixed. I can follow up.

**Pull request merged**